### DISCUSSION TITLE
Replace popperjs by floating-ui

### DISCUSSION THREAD

**Discussion by nclemeur** [vueform/vueform#135] [Ideas]
Looks like floating-ui is the successor of popper...

**Comment by nclemeur** [comment]
Might actually be good to have a look at supporting popover instead https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover. Most browser support it (or will soon). Looks like that was the approach taken by formkit

**Reply by adamberecz** [reply]
Popover will be a great option for managing dialogs, modals, etc. because it seems to resolve position/overlay issues. However with dropdowns the challenge is that the user should be able to scroll the page while open, otherwise we have this effect as in Formkit's implementation:
![formkit-popover-2](https://github.com/vueform/vueform/assets/16024069/480c41ee-38ed-43e7-949b-cc8ea45424ab)
*(Google Chrome - Version 121.0.6167.184)*

But it's true that `floating-ui` is the successor of `popperjs` and might save a few KBs if replaced.
